changelog.txt

```diff
@@ -1,3 +1,6 @@
+1.7.0
+    - refactor milestone signature parameters to be configurable (#1322)
```

Comment:
add the Coo change itself

Reply:
migration from legacy coordinator to compass

Sounds good?

Reply:
yep